src/mozanalysis/experiment.py

```diff
@@ -637,14 +650,18 @@ def _build_enrollments_query_normandy(self, time_limits: TimeLimits) -> str:
             AND e.submission_date
                 BETWEEN '{first_enrollment_date}' AND '{last_enrollment_date}'
             AND e.event_string_value = '{experiment_slug}'
+            AND e.sample_id <= {sample_size}
         GROUP BY e.client_id, branch
             """.format(
             experiment_slug=self.experiment_slug,
             first_enrollment_date=time_limits.first_enrollment_date,
             last_enrollment_date=time_limits.last_enrollment_date,
+            sample_size=sample_size or 100,
```

Comment:
Why the "default none" and "sample_size or 100" pattern instead of setting a default value for the parameter in the function definition?

Reply:
I actually had it the other way, but noticed that this sort of pattern was used in one of the other enrollment functions. I'm OK with setting the default via function signature though, I'll change it.

Reply:
I don't know if I have an opinion, I was just curious